glad/parse.py

```diff
@@ -1,3 +1,4 @@
+from tkinter import N
```

Comment:
Leftover? Definitely don't want to depend on tkinter

Reply:
Yeah, probably auto imported by my ide haha

Reply:
Yeah, probably auto-imported by my ide lol